src/routes/AppContent/index.tsx

```diff
@@ -67,6 +68,40 @@ const AppContent = () => {
         setActiveTab(getActiveTab(location.pathname));
     }, [location]);
 
+    const isP2pBlocked = activeAccountData && (activeAccountData.is_virtual || activeAccountData.currency !== 'USD');
+
+    const getComponent = () => {
+        let content = null;
+
+        if ((isLoadingActiveAccount || !isFetched || !activeAccountData) && !isEndpointRoute) {
+            content = <Loader />;
+        } else if (isP2pBlocked && !isEndpointRoute) {
+            content = <BlockedScenarios type={getBlockedType(activeAccountData) ?? ''} />;
+        } else if (isFetched && activeAccountData) {
+            content = (
+                <div className='app-content__body'>
+                    <Tabs
+                        activeTab={activeTab}
+                        className='app-content__tabs'
+                        onChange={index => {
+                            setActiveTab(tabRoutesConfiguration[index].name);
+                            history.push(tabRoutesConfiguration[index].path);
+                        }}
+                        variant='secondary'
+                    >
+                        {tabRoutesConfiguration.map(route => (
+                            <Tab key={route.name} title={route.name} />
+                        ))}
+                    </Tabs>
+                    {isDesktop && !isEndpointRoute && <GuideTooltip />}
+                    <Router />
+                </div>
+            );
+        }
+
+        return content;
+    };
```

Comment:
instead of storing the component inside of a temp variable, we can make this slightly more efficient and readable by just directly returning the component. This way, it doesn't just iterate through the whole if statement to finally return the component, and it will just return it whenever the condition was satisfied

```suggestion
    const getComponent = () => 
        if ((isLoadingActiveAccount || !isFetched || !activeAccountData) && !isEndpointRoute) {
            return <Loader />;
        } else if (isP2pBlocked && !isEndpointRoute) {
            return <BlockedScenarios type={getBlockedType(activeAccountData) ?? ''} />;
        } else if (isFetched && activeAccountData) {
            return (
                <div className='app-content__body'>
                    <Tabs
                        activeTab={activeTab}
                        className='app-content__tabs'
                        onChange={index => {
                            setActiveTab(tabRoutesConfiguration[index].name);
                            history.push(tabRoutesConfiguration[index].path);
                        }}
                        variant='secondary'
                    >
                        {tabRoutesConfiguration.map(route => (
                            <Tab key={route.name} title={route.name} />
                        ))}
                    </Tabs>
                    {isDesktop && !isEndpointRoute && <GuideTooltip />}
                    <Router />
                </div>
            );
        }

        return null;
    };
```